lib/selector/parser/ast.go

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
```

Comment:
I see no need for the length of this filename to be restricted to three letters.